cpp/lib/src/Zotero.cc

```diff
@@ -160,6 +160,36 @@ bool Web(const Url &zts_server_url, const TimeLimit &time_limit, Downloader::Par
 } // namespace TranslationServer
 
 
+template <typename T>
```

Comment:
Please put this template into JSON.h, name it VisitLeafNodes, and start it with "template<typename ParamsType, typename CallbackType>".